fire/cli/niv/regn.py

```diff
@@ -217,19 +218,26 @@ def gama_beregning(
         )
 
     # Lad GNU Gama om at køre udjævningen
+    if kontrol:
+        beregningstype = "kontrol"
+    else:
+        beregningstype = "endelig"
+
     ret = subprocess.run(
         [
             "gama-local",
             f"{projektnavn}.xml",
             "--xml",
             f"{projektnavn}-resultat.xml",
             "--html",
-            f"{projektnavn}-resultat.html",
+            f"{projektnavn}-resultat-{beregningstype}.html",
         ]
     )
     if ret.returncode:
-        fire.cli.print(f"Check {projektnavn}-resultat.html", bg="red", fg="white")
-    webbrowser.open_new_tab(f"{projektnavn}-resultat.html")
+        fire.cli.print(
+            f"Check {projektnavn}-resultat-{beregningstype}.html", bg="red", fg="white"
+        )
+    webbrowser.open_new_tab(f"{projektnavn}-resultat-{beregningstype}.html")
 
     # Grav resultater frem fra GNU Gamas outputfil
     with open(f"{projektnavn}-resultat.xml") as resultat:
```

Comment:
Ellers skal der også ændres her

Reply:
Jes I know, men så vidt jeg kan se bruges den kun til at hive resultaterne frem, så de kan puttes i et excel-ark.. Faktisk ved jeg ikke helt hvad vi skal bruge dem til sådan fremadrettet..? Står der noget i dem, som IKKE kommer i excel-filen eller html-filen?

Reply:
Muligvis - men vi bruger det ikke til noget p.t. Jeg spurgte bare for at være sikker på at det var intentionen.